base/testfiles-legacy/github-0730.lvt

```diff
@@ -0,0 +1,12 @@
+\documentclass{article}
+
+\input{test2e}
+\DeclareUnicodeCharacter{009C}{\fbox{U+009C}}
+
+
+\begin{document}
+
+\START
+% This should show the fbox (and not error)
+\setbox0\hbox{U+009C:}\showbox0
+\END
```

Comment:
I would be very surprised if this shows and error looking at it :-) but I guess the file actually contains the offending invisible characters. Difficult to tell in the github interface

Reply:
maybe be it would be better to show the character actually as the two bytes it is represented with (`^^c2^^9c`), because even looking with an editor at this file in the future will result in some head scratching I guess